samples/languages/dotnet/README.md

```diff
@@ -1,6 +1,23 @@
 Running a DotNet application using SGX-LKL-OE
 =============================================
 
+It is possible to run this sample by typing:
+
+```
+make run-hw
+```
+
+or
+
+```
+make run-sw
+```
+
```

Comment:
Do we normally give two options with no explanation of which you should do? If yes, then I think this is ok, because it isn't worse than what we usually do, however, unless someone knows what run-hw and run-sw do, this provides no context for picking one.

I think it should have a short explanation for each "run with SGX on hardware that supports it" or "run emulated". Something to that effect.

Reply:
Other samples are as vague as this, and I didn't want to spend too much time on changing readmes in this PR.

I think this should be done as part of https://github.com/lsds/sgx-lkl/issues/332, which is a bigger task. It would be nice to get rid of the Makefiles, or to make then easier to understand. But, definitely, all samples need to be cleaned up a bit.